pyCRLD/Environments/MultipleObsSocialDilemma.py

```diff
@@ -0,0 +1,127 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../../nbs/Environments/12_MultipleObsSocialDilemma.ipynb.
```

Comment:
This file simply implements the social dilemma layer into the heterogeneous observation env. file.
I've initially tried to incorporate the "contract" idea because I saw this in the Uncertain Environment file, however, I don't really understand the dynamics of contract and I don't think it's fully functional, I need to work on it.
I thought it wasn't relevant for our project as IPD only has the one state `.`. Please let me know if I misunderstood this.